clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -417,18 +417,15 @@ describe("Consent banner", () => {
         });
 
         cy.contains("button", "Manage preferences").click();
+        cy.wait("@getPrivacyExperience");
 
         // Save new preferences
-        cy.getByTestId("toggle-Test privacy notice").click();
-        cy.getByTestId("toggle-Essential").within(() => {
-          cy.get("input").should("be.disabled");
-        });
-        cy.getByTestId("Save test-btn").click();
+        cy.getByTestId("toggle-Advertising").click();
```

Comment:
To explain these cypress changes- Cypress essentially uses the pre-fetch flow, where we initially stub the experience API using `consent/test_banner_options.json`. In this PR, we've adjusted the pre-fetch flow where we make a Fides BE call if any notices in pre-fetched experience do not have user consent. Our `consent-banner.cy.ts` actually uses a different mock (`consent/overlay_experience.json`) when we make any additional calls out to Fides BE, so that's why the expected results change here, and I've had to make those updates in the tests.